src/devtools/client/debugger/src/components/FullTextSearch/index.js

```diff
@@ -69,7 +68,7 @@ export class FullTextSearch extends Component {
       return;
     }
 
-    const query = sanitizeQuery(e.target.value);
+    const sanitizedQuery = sanitizeQuery(e.target.value);
```

Comment:
Is this an important sanitization? If so we should probably do it on the backend.

Reply:
that's fair